kitchen.yml

```diff
@@ -3,10 +3,12 @@ driver:
   name: vagrant
   customize:
     cpus: 2
-    memory: 4096
-    differencing_disk: true
-  network:
-    - ["public_network", {bridge: "Default Switch"}]
+    memory: 2048
+    clipboard: bidirectional
+    draganddrop: bidirectional
+#    differencing_disk: true
+#  network:
+#    - ["public_network", {bridge: "Default Switch"}]
```

Comment:
this was me for hyper-v, could you just remove this for now please? :)

Reply:
will do